### DISCUSSION TITLE
New wallet: a wallet developed by Oasis Network

### DISCUSSION THREAD

**Discussion by btcdca8** [oasisprotocol/community-forum#19] [Ideas]
It is recommended that the Oasis Network team can develop a wallet instead of relying on a third-party wallet. The development of Oasis Wallet can better connect with defi applications.The ecology will be even bigger!

**Comment by tjanez** [comment]
Thanks for the feedback.

I can confirm we are developing an official Oasis Wallet in the form of a Chrome Extension.

We'll keep you posted!

**Comment by Esya** [comment]
I'm currently building a web wallet that will be fully open-source and self-custodian, which will I think release in about a month, and maybe in 6-8 weeks with staking & ledger support - this is part of the [https://github.com/oasisprotocol/community/discussions/12](ROSE Bloom Grants) program 

It will support using private keys directly, mnemonics, and ledger

![n8bG1p6 (1)](https://user-images.githubusercontent.com/937179/114441313-83fd1600-9bcb-11eb-8cd2-4de50fecefcd.gif)

**Comment by tjanez** [comment]
This has been done.

We launched two official Oasis Wallets, the web and the browser extension variants, on Sep 22, 2021.

To use the web variant, browse to: https://wallet.oasisprotocol.org/

To use the browser extension variant, install it from the [Chrome Web Store](https://chrome.google.com/webstore/detail/oasis-wallet/ppdadbejkmjnefldpcdjhnkpbjkikoip).

More details:
- Documentation: https://docs.oasis.dev/general/manage-tokens/oasis-wallets
- Code:
  - Web: https://github.com/oasisprotocol/oasis-wallet-web
  - Browser Extension: https://github.com/oasisprotocol/oasis-wallet-ext
